### DISCUSSION TITLE
Player claim data

### DISCUSSION THREAD

**Discussion by ramanzxo** [GriefPrevention/GriefPrevention#2120] [Q&A]
my players change their mc name but the server is crack enable so all the data needs to be transfer to the new mc name , but how can i easily find the player claim data without having to single handily go through all the claim data? please help me because theres over 300+ claim data to look. Thankyou

**Comment by Jikoo** [comment]
Find their old UUID, then move the file for their old UUID to the corresponding new UUID.
